### DISCUSSION TITLE
Dense upsamling operation rather than deconvolution implementation

### DISCUSSION THREAD

**Discussion by chowkamlee81** [apache/incubator-mxnet#7907] [Q&A]
I developed a small snippet to replace deconvolution by Dense-up sampling operation according to mxnet example  incubator-mxnet/example/gluon/super_resolution.py.

My i/p is `1024*48*64`
My o/p required is `19*768*1024`.

Hence i did conv to produce `(19*16*16)*48*64`. Here upsampling factor is 16, num_classes=19
Through your technique i got `19*768*1024` which is desired accordingly when i ran using jupyter notebook graphically diagram of network.
But when i give this symbol for training, exception thrown below
Kindly suggest and help pls

" src/operator/./crop-inl.h:139: Check failed: param_.offset[1] <= data_shape[3]-out_shape[3] (8 vs. 0) offset[1] should be less than the residual space of width"

**Comment by szha** [comment]
This looks like that the error comes from center crop op. Since the input size is different, you need to adjust that piece of code, especially the crop-size, accordingly.

**Comment by chowkamlee81** [comment]
Hai szha,

I tried according to you and i removed crop function as you suggested. After that module started training. Even after several epochs training log loss measure remains same after several epochs.

Below is report
Epoch[0] Batch [10] Speed: 3.88 samples/sec Train-FCNLogLoss= 2.944441
.............................................

Epoch[0] Batch [730] Speed: 3.843 samples/sec Train-FCNLogLoss= 2.944441.

Kindly suggest how to go ahead and solve this issue

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.

**Comment by leleamol** [comment]
Recommended Labels: "Question"

**Comment by anirudhacharya** [comment]
@chowkamlee81 are you still facing this issue. If yes, can you share the code snippet that is producing this error?

**Comment by anirudhacharya** [comment]
@mxnet-label-bot remove [Example]
